test/models/generative_models.jl

```diff
@@ -8,10 +6,12 @@ ys = counterfactual_data.y
 generator = generator_catalogue[:revise]()
```

Comment:
the catalogue could be referenced from Generators I believe

Reply:
Oh right, forgot that the catalogue comes from there